### PR TITLE
[GRIFFIN-273] Griffin Service with user-defined log4j2 configuration 

### PR DESCRIPTION
**Pull request by joohnnie**
Currently, the Griffin service module use spring-boot default log4j2 configuration.

This approach is not easy to change the config for debugging and add new appender such as 
RollingFile

### CODE REVIEW

**Comment by guoyuepeng** [comment]
hi Joohnnie, 
Could you rebase, some commits are already merged before.

**Comment by joohnnie** [comment]
Hi William,
I have rebased and please help to check.

Thanks
Johnnie

**Comment by guoyuepeng** [comment]
Thanks, Johnnie.

**Comment by whhe** [comment]
It seems that your configuration is based on the default configuration of spring boot, I think there is no problem with the content. 

But I'm not sure if giving an implementation like what you do or giving a guide in a document is better, cause If users want customized log, they may still need to make some changes in these files.

**Comment by joohnnie** [comment]
@whhe Thanks for your comments. 
The config is the default configuration from spring boot. The reason I put it to the code base because we can easily change the config based on our need. Otherwise, users even don't know what' the log library they used.
In addition to that, I removed some dependencies as they will cause the confliction in the log system.

Regarding the guild, log4j2 is an open-source library and has its official website.
User can refer to the website to customize the config.  
https://logging.apache.org/log4j/2.x/index.html

File: service/pom.xml [hunk 1]
```diff
@@ -82,6 +82,10 @@ under the License.
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
```
Comment (guoyuepeng) [inline service/pom.xml]: LGTM.

**Comment by guoyuepeng** [comment]
LGTM.

**Pull request closed without merging**